test-analysis-core/src/main/kotlin/com/saveourtool/save/test/analysis/algorithms/FlipRateAnalysis.kt

```diff
@@ -0,0 +1,42 @@
+package com.saveourtool.save.test.analysis.algorithms
+
+import com.saveourtool.save.test.analysis.api.TestRuns
+import com.saveourtool.save.test.analysis.api.metrics.RegularTestMetrics
+import com.saveourtool.save.test.analysis.api.results.FlakyTest
+import com.saveourtool.save.test.analysis.api.results.IrregularTest
+
+/**
+ * _Flip-rate_ based flaky test detection algorithm.
```

Comment:
can you please add more information about algorithm here?

Reply:
Addressed with 7607e24e9dd1034bc245dc517f53154d79d8aa5e.